test/sources/cryptography.js

```diff
@@ -311,7 +313,7 @@ describe('Check cryptography', function () {
         '711f87668175afae5f158a4f16ec705e558b8f7c9e494f12534759453bbfa004',
       '84e0d4ae17ceefd457da118729539d121c9f5586f82338d895d1744652ce4455':
         '069714edac1fdb7f932f7f0af657f19982b6e72318781e042d25cea12311086c',
-      'f5864ab6a5a2190666b47c676bcf15a1f2f07703c5bcafb5749aa735ce8b7c36':
+      f5864ab6a5a2190666b47c676bcf15a1f2f07703c5bcafb5749aa735ce8b7c36:
```

Comment:
Nit: This change looks inconsistent. Could it be switched off with `eslint-disable-next-line`?